### DISCUSSION TITLE
Is this script help me to create own SMTP server?

### DISCUSSION THREAD

**Discussion by shanmugad** [PHPMailer/PHPMailer#2855] [Q&A]
Hi

Can i create own SMTP Server with this PHPMailer? (Without using any external SMTP Server) 

or if you know any PHP Open source or script which help to create SMTP SERVER. Please mention

**Comment by Synchro** [comment]
No, PHPMailer won't help you with that. It provides a basic SMTP client, but that's all. I would not recommend writing an SMTP server in PHP anyway - mail servers are very tricky things, and there are quite a lot of very good open-source ones (qmail, postfix, exim), and some more unusual ones like Haraka (written in Javascript), so I would recommend you use one of those rather than trying to do something in PHP.

**Reply by shanmugad** [reply]
@Synchro 

Thanks for answering me here and stackoverflow. I know many open source like Modboa, iRedmail, etc. I have one server in Digital ocean which have WordPress running. 

To install new Email server application, then i have to buy new VPS and which increase my cost and i am not affordable at this moment. 

So, the one which running WordPress have presetup of apache, mySQL, PHP, etc. So, i thought of any SMTP open source or script in PHP. 

it will save my money on long run. i can put that script in same server.

**Reply by Synchro** [reply]
What's stopping you from installing postfix on your Wordpress server? There's no need to get another VPS just to run it.
